tests/web/rpc/v1/test_endpoint_trace_item_table/test_endpoint_trace_item_table.py

```diff
@@ -3527,3 +3529,56 @@ def test_non_existant_attribute_filter(self) -> None:
                 ],
             ),
         ]
+
+    def test_cached_query_results_is_used_in_routing(self) -> None:
```

Comment:
I don't think you need to keep this test in here. Thank you for doing it to make sure that the caching keeps the `elapsed` field intact. 

I would recommend removing this test because it's too dependent on internal details of the endpoint